documentation/modules/configuring/con-config-mirrormaker2-tasks-max.adoc

```diff
@@ -16,9 +17,19 @@ If there are more tasks than workers, workers handle multiple tasks.
 
 You can specify the maximum number of connector tasks in your MirrorMaker configuration using the `tasksMax` property.
 Without specifying a maximum number of tasks, the default setting is a single task.
-If the infrastructure supports the processing overhead, increasing the number of tasks can improve throughput.
 
-.tasksMax configuration for a MirrorMaker connector
+The heartbeat connector always uses a single task.
+
+The number of tasks that are started for the source and checkpoint connectors is the lower value between the maximum number of possible tasks and the value for `tasksMax`.
+For the source connector, the maximum number of tasks possible is one for each partition being replicated from the source cluster.
+For the checkpoint connector, the maximum number of tasks possible is one for each consumer group being replicated from the source cluster.
```

Comment:
What are the _possible tasks_ here? What determines how many tasks are possible? Do we explain it somewhere?

Reply:
We're suggesting that the max possible is the lower value of one for each partition or the value set for `tasksMax` value, which defaults to 1. 
But, as Kafka suggest, the value used depends _on the available hardware resources and the total number of topic-partitions to be replicated._